addons/binding/org.openhab.binding.velux/doc/allclasses-frame.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
If this is generated javadoc, please leave it away

Reply:
Eliminated.